coordinator/src/routes/health/get-status.ts

```diff
@@ -11,21 +12,45 @@ export interface StatusCheckResponse {
   links?: string
 }
 
-export async function serviceHealthCheck(url: string): Promise<StatusCheckResponse> {
-  return await axios
-    .get<string>(url, {timeout: 20000})
-    .then((response): StatusCheckResponse => ({
+export async function serviceHealthCheck(url: string, logger: pino.Logger): Promise<StatusCheckResponse> {
+  try {
+    const response = await axios.get<string>(url, {timeout: 20000})
+    return {
       status: response.status === 200 ? "pass" : "error",
       timeout: "false",
       responseCode: response.status,
       outcome: response.data,
       links: url
-    }))
-    .catch(error => ({
+    }
+  } catch (error) {
+    logger.error(error)
```

Comment:
Might be a good idea to add a log line for some context, e.g. `logger.error(error, "Error calling external service for status check")`

Reply:
done